.eslintrc.json

```diff
@@ -277,6 +277,7 @@
 						// imports that are allowed in all /test/ files
 						"when": "test",
 						"allow": [
+							"vs/css.build",
```

Comment:
expected that this only has `vs/css.build` and not others from this move?

Reply:
Yes, I've added a test which uses `vs/css.build`. `vs/nls` is allowed in all layers and is handled in the rule itself, while `vs/css` is allowed in all browser layers and is handled a bit above.